Mathlib/Algebra/Star/Order.lean

```diff
@@ -244,6 +244,28 @@ theorem mul_star_self_pos [Nontrivial R] {x : R} (hx : IsRegular x) : 0 < x * st
 
 end NonUnitalSemiring
 
+section NonUnitalRing
+
+variable [NonUnitalRing R] [PartialOrder R] [StarRing R] [StarOrderedRing R]
+
+lemma IsSelfAdjoint.mul_nonneg_mul {a b : R} (ha : IsSelfAdjoint a) (hb : 0 ≤ b) :
+    0 ≤ a * b * a := by
```

Comment:
Note that I also have that one in #16050. I think I slightly prefer `conjugate` in the name like we have elsewhere but it's not a strong preference.

Reply:
closing in favor of #16050, we can just add these there.